### PR TITLE
ae-424 +ae-425

### PR DESCRIPTION
**Pull request by tanfarming**
## References

JIRA:
- https://mozilla-hub.atlassian.net/browse/AE-424
- https://mozilla-hub.atlassian.net/browse/AE-425

## Description
- Add region dropdowns for all countries on the Preview page
- Make ads on preview page clickable

## PR Review Checklist

_Put an `x` in the boxes that apply_

- [ ] This PR conforms to the [Contribution Guidelines](https://github.com/mozilla-services/consvc-shepherd/blob/main/CONTRIBUTING.md).
- [ ] The PR title starts with the JIRA issue reference, format `[DISCO-####]`, and has the same title (if applicable).
- [ ] [Documentation](https://github.com/mozilla-services/consvc-shepherd/tree/main/docs) has been updated (if applicable).
- [ ] Functional and performance test coverage has been expanded and maintained (if applicable).

### CODE REVIEW

File: consvc_shepherd/preview.py [hunk 1]
```diff
@@ -147,76 +151,29 @@ class Ads:
...
-        Region(code="SK", name="Saskatchewan"),
-        Region(code="NT", name="Northwest Territories"),
-        Region(code="NU", name="Nunavut"),
-        Region(code="YT", name="Yukon"),
-    ],
-}
+
+def load_regions() -> dict[str, list[Region]]:
+    """Load Regions"""
+    with open("./static/preview/iso-3166-2.json", "r") as file:
```
Comment (cbguder) [inline consvc_shepherd/preview.py:157]: Static files are a bit finicky in Django apps. I think it's worth double checking this file is where you expect it to be in the dev environment (or at least a local docker build).

File: consvc_shepherd/preview.py [hunk 2]
```diff
@@ -287,10 +246,13 @@ def get_amp_tiles(env: Environment, country: str, region: str) -> list[Tile]:
         f"{env.mars_url}/v1/tiles", params=params, headers=headers, timeout=30
     )
 
+    print("Tiles: ", r.json().get("tiles", []))
```
Comment (cbguder) [inline consvc_shepherd/preview.py:249]: Stray print

File: templates/preview.html [hunk 3]
```diff
@@ -45,21 +45,25 @@ <h1>MARS Ads Preview</h1>
     <h2>Tiles</h2>
     <div class="tiles">
         {% for tile in ads.tiles %}
+        <a href="{{ tile.url }}" target="_blank">
```
Comment (cbguder) [inline templates/preview.html:48]: Can you make sure this doesn't change the styling on the page (e.g. doesn't add underlines?)

File: consvc_shepherd/preview.py (hunk 1 above)
Comment (tanfarming) [reply consvc_shepherd/preview.py:157]: [everything in ./static is copied over to /app/static](https://github.com/mozilla-services/consvc-shepherd/blob/5ac956730f5d9ec3f7b67bfd2373448cd898cab3/Dockerfile#L35C1-L35C28)
and [workdir is set to /app](https://github.com/mozilla-services/consvc-shepherd/blob/5ac956730f5d9ec3f7b67bfd2373448cd898cab3/Dockerfile#L18) 
i verified it inside the container and tested it locally with `docker-compose up --build` everything seems to be working, 
i'm feeling pretty safe here lol but let me know if i missed anything

File: consvc_shepherd/preview.py (hunk 2 above)
Comment (tanfarming) [reply consvc_shepherd/preview.py]: yep, thx, found another one and fixed it too

File: templates/preview.html (hunk 3 above)
Comment (tanfarming) [reply templates/preview.html]: done

Comment (tanfarming) [reply templates/preview.html]: fixed

**Comment by tanfarming** [comment]
closing this one to submit a new pr to include mobile and debugging info features

**Pull request closed without merging**